### ISSUE TITLE
xosc VisibilityAction schema error, 'traffic' is replaced by 'active'

### ISSUE THREAD

**Issue by 31415tertje** [pyoscx/scenariogeneration#7]
When validating the generated Visibility Action: VisibilityAction(graphics=False, traffic=False, sensors=False)
an error is given:
`failed validating {'active': 'false', 'graphics': 'false', 'sensors': 'false'} with XsdAttributeGroup(['graphics', 'sensors', 'traffic']):

Reason: missing required attribute: 'traffic'

Schema:

  <xsd:complexType xmlns:xsd="http://www.w3.org/2001/XMLSchema" name="VisibilityAction">
      <xsd:attribute name="graphics" type="Boolean" use="required" />
      <xsd:attribute name="sensors" type="Boolean" use="required" />
      <xsd:attribute name="traffic" type="Boolean" use="required" />
  </xsd:complexType>

Instance:

  <VisibilityAction active="false" graphics="false" sensors="false" />

Path: /OpenSCENARIO/Storyboard/Story/Act/ManeuverGroup[1]/Maneuver/Event/Action/PrivateAction/VisibilityAction`

I don't know if it is deliberate, but the get_attributes returns 'active' in stead of 'traffic'.

File xosc/actions.py
```
def __init__(self,graphics, traffic, sensors):
        if not isinstance(graphics,bool):
            raise TypeError('graphics input is not of type bool')
        if not isinstance(traffic,bool):
            raise TypeError('traffic input is not of type bool')
        if not isinstance(sensors,bool):
            raise TypeError('sensors input is not of type bool')
        self.graphics = graphics
        self.traffic = traffic
        self.sensors = sensors


def get_attributes(self):
        return {'graphics':convert_bool(self.graphics),'active':convert_bool(self.traffic),'sensors':convert_bool(self.sensors)}
```

**Comment by mander76** [comment]
Yes, this is a bug, (probably a copy paste or something. Will fix

**Comment by mander76** [comment]
Bug, fixed and will be in the next release (probably by the end of the week)

**Comment by mander76** [comment]
fixed in release 0.3.0
